pkgs/development/python-modules/langchain-ollama/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  poetry-core,
+
+  # dependencies
+  langchain-core,
+  ollama,
+
+  nix-update-script,
+}:
+
+buildPythonPackage rec {
+  pname = "langchain-ollama";
+  version = "0.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "langchain-ai";
+    repo = "langchain";
+    rev = "refs/tags/langchain-ollama==${version}";
+    hash = "sha256-NtlJqIevdaoOO4oDqKsL4OFbw8tHb3FjV9LrTytOGKE=";
+  };
+
+  sourceRoot = "${src.name}/libs/partners/ollama";
+
+  build-system = [ poetry-core ];
+
+  dependencies = [
+    langchain-core
+    ollama
+  ];
+
+  pythonImportsCheck = [ "langchain_ollama" ];
+
```

Comment:
Can we enable unit tests?

Reply:
Unfortunately, they don't supply any tests.

Reply:
How about this https://github.com/langchain-ai/langchain/tree/master/libs/partners/ollama/tests ?

Reply:
*sigh* I was looking in the wrong place. Working on it.

Reply:
Here you go.